core/server/api_container/server/startosis_engine/kurtosis_instruction/run_sh/run_sh.go

```diff
@@ -205,14 +206,14 @@ func (builtin *RunShCapabilities) Execute(ctx context.Context, _ *builtin_argume
 	}
 
 	// run the command passed in by user in the container
-	code, output, err := builtin.serviceNetwork.ExecCommand(ctx, builtin.name, createDefaultDirectory)
+	createDefaultDirectoryResult, err := builtin.serviceNetwork.RunExec(ctx, builtin.name, createDefaultDirectory)
 	if err != nil {
 		return "", stacktrace.Propagate(err, fmt.Sprintf("error occurred while executing one time task command: %v ", builtin.run))
 	}
 
 	result := map[string]starlark.Comparable{
-		runshOutputKey: starlark.String(output),
-		runshCodeKey:   starlark.MakeInt(int(code)),
+		runshOutputKey: starlark.String(createDefaultDirectoryResult.GetOutput()),
```

Comment:
I suppose we can add `GetStdError` or something to the execResult obj in the future or pull it from there into starlark

Reply:
Oh yes you are right! I have seen the red out put at the end. We don't print new lines properly on it and I see `\n` but I do see the errors. 

I'll create a ticket for the exec error being a bit messy (new lines not rendered properly) separately

Reply:
Yup we can track that separately, shouldn't be too difficult to fix, hopefully